src/main.js

```diff
@@ -55,14 +56,115 @@ import { TrajectoryCalculator } from "./core/trajectoryCalculator.js";
     let playerTurn = true;
     let [playerOneMoveDist, playerTwoMoveDist] = [20, 20];
 
+    // creating the 'world' object to do physics calculations
+    let world = new World();
+ 
+    world.setGravity(Vec2(0, -9.8));
+
+    const sf = 25; // scale factor to scale metric unit system of planckjs to pixijs pixel system
+    // need to convert from planck.js coord sys to pixijs coord sys, and back
+    function convertPlanckYToPixiY(planckY) {
+        return (app.renderer.height - (planckY * sf));
+    }
+
+    function convertPlanckXtoPixiX(planckX) {
+        return (planckX * sf);
+    }
+
+    function convertPixiXtoPlanckX(pixiX) {
+        return pixiX / 25;
+    }
+
+    function convertPixiYToPlanckY(pixiY) {
+        return (app.renderer.height - pixiY) / sf;
+    }
+
+    function convertDegreesToRadians(degrees) {
+        return degrees * (Math.PI / 180); 
+    }
+
+    // this will keep track of the bullet sprite, as well as its corresponding 
```

Comment:
corresponding what?

Reply:
Corresponding body Planck object, not sure why it didn't save